targets/ipq806x

```diff
@@ -1,5 +1,3 @@
-ATH10K_PACKAGES=
```

Comment:
Please keep the initialization, all variables should explicitly be set to an empty string.